xml/deployment_bootoptions.xml

```diff
@@ -579,7 +549,11 @@ The normal installation mode.
      <listitem>
       <para>
        The default value is <literal>dhcp</literal> which configures the
-       network via DHCP. With the option
+       network via DHCP. Set <literal>netsetup=-dhcp</literal> when
+       configuring the network with with the boot parameters
+       <literal>hostip</literal>, <literal>gateway</literal> and
+       <literal>nameserver</literal>.
+       With the option
        <literal>netsetup=hostip,netmask,gateway,nameserver</literal> the
        installer asks for the network configuration during boot.
       </para>
```

Comment:
This sounds weird and mixes things. 

* `netsetup=dhcp` -> will set up a network with DHCP even if linuxrc does not need it (e.g. when doing an installation from DVD
* `netsetup=hostip,netmask,gateway,nameserver` -> will prompt for the requested values and also set up a network in any case

See https://en.opensuse.org/SDB:Linuxrc#p_netsetup for more information

`hostname=` / `hostip=` / ... will only setup a network when _linuxrc_ needs it (e.g. when doing a remote installation). It will _not_ set up the network when doing an installation from a local image or DVD, for example. This behavior is _not_ something the customer expects! Also only sets up the network for the first interface that was found.

With `ifcfg=` you can configure networks on different interfaces. This is also the only way to configure VLANs.

See https://en.opensuse.org/SDB:Linuxrc#Network_Config for details on this topic.